pkg/sdk/common/opaque.go

```diff
@@ -41,6 +41,26 @@ func DecodeOpaqueMap(opaque *types.Opaque) map[string]string {
 	return entries
 }
 
+// EncodeOpaqueMap encodes a map of strings into a Reva opaque entry.
+// Only plain encoding is currently supported.
+func EncodeOpaqueMap(opaque *types.Opaque, m map[string]string) {
```

Comment:
`rename to EncodePlainOpaqueMap` or accept a decoder argument. This assumes too much.

Reply:
The latter. Done in https://github.com/cs3org/reva/pull/2383